### PR TITLE
Implement parseQuery

### PR DESCRIPTION
**Pull request by jwicks31**
resolves #112

### CODE REVIEW

**Comment by kennylavender** [comment]
because I created an issue for this specific function, the pr should now resolve that issue. I am updating your description.

File: src/parse-query.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+import words from 'lodash/words';
+export const parseQuery = ({ ft } = {}) => words(ft, /[^,]+/g);
```
Comment (kennylavender) [inline src/parse-query.js]: regex and `words` is a bit overkill here, lets change this to [split](https://lodash.com/docs/4.17.10#split) instead.

Comment (jwicks31) [reply src/parse-query.js]: Ok. I thought it might be.

File: src/parse-query.js [hunk 2]
```diff
@@ -1,2 +1 @@
-import words from 'lodash/words';
-export const parseQuery = ({ ft } = {}) => words(ft, /[^,]+/g);
+export const parseQuery = ({ ft } = {}) => (ft ? ft.split(',') : []);
```
Comment (kennylavender) [inline src/parse-query.js]: This is fine, but did lodash's split not work?

**Pull request merged**